### ISSUE TITLE
Performance Metric Missing

### ISSUE THREAD

**Issue by Bearwithchris** [rinongal/StyleGAN-nada#71]
Hi, Would it be possible to share with us the code for the performance metrics. 
Specifically LPIPS used for generated data, as mentioned in the paper "As our own method does not use a training set, we cluster around generated images using K-Medoids [1]."?

**Comment by rinongal** [comment]
Hi,

Unfortunately it's been almost 3 years and I don't have access to the machine with the original code.

We just used the clustering metric code from [Ojha et al](https://github.com/utkarshojha/few-shot-gan-adaptation/blob/main/feat_cluster.py) except that we use [sklearn's K-Medoids](https://scikit-learn-extra.readthedocs.io/en/stable/generated/sklearn_extra.cluster.KMedoids.html) to get the points from which we calculate distances (their code assumes you have a training set to calculate distances to).

**Comment by Bearwithchris** [comment]
Thank you for the prompt reply. No worries, I would then like to clarify the procedure to attain the centroids. Specifically, I have the following questions:

1. In what latent/pixel space is K-medoids applied on?  
2. Then given that [sklearn's K-Medoids](https://scikit-learn-extra.readthedocs.io/en/stable/generated/sklearn_extra.cluster.KMedoids.html) works on 2-dim data, is some kind of dimensions reduction applied to the latent/pixel space prior to clustering? 
3. Lastly, if some dimensional reduction was applied, how did you get the "gen-image" pertaining to the centroid for LPIPS measurements?
